<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>474</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>-1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* --- GENEL AYARLAR --- */
QWidget {
    font-family: &quot;Segoe UI&quot;, &quot;Calibri&quot;, sans-serif;
    font-size: 14px;
    color: #333333;
}

/* --- BUTONLAR --- */
QPushButton {
    background-color: #2980b9; /* Kurumsal Mavi */
    color: white;
    border-radius: 5px;
    padding: 8px 15px;
    font-weight: bold;
    border: none;
}
QPushButton:hover {
    background-color: #3498db; /* Üzerine gelince açıl */
}
QPushButton:pressed {
    background-color: #1f618d; /* Basınca koyulaş */
}
/* &quot;SİL&quot; Butonlarını Kırmızı Yapmak İstersen Özel İsimle Yakala */
QPushButton#btn_sil, QPushButton#btn_satir_sil {
    background-color: #c0392b;
}
QPushButton#btn_sil:hover, QPushButton#btn_satir_sil:hover {
    background-color: #e74c3c;
}

/* --- GİRİŞ KUTULARI (LineEdit, DateEdit, ComboBox) --- */
QLineEdit, QComboBox, QDateEdit, QTimeEdit {
    border: 1px solid #bdc3c7;
    border-radius: 4px;
    padding: 5px;
    background-color: white;
}
/* İçine tıklayınca (Focus) çerçeve parlasın */
QLineEdit:focus, QComboBox:focus, QDateEdit:focus {
    border: 2px solid #3498db;
}

/* --- TABLOLAR (En Önemlisi!) --- */
QTableWidget {
    background-color: white;
    gridline-color: #ecf0f1; /* Hücre çizgileri silik gri */
    border: 1px solid #bdc3c7;
}
/* Başlık Satırı */
QHeaderView::section {
    background-color: #2c3e50; /* Koyu Lacivert Başlık */
    color: white;
    padding: 5px;
    border: none;
    font-weight: bold;
}
/* Seçili Satır */
QTableWidget::item:selected {
    background-color: #d6eaf8; /* Açık mavi seçim */
    color: black;
}

/* --- GROUP BOX (Çerçeveler) --- */
QGroupBox {
    border: 2px solid #bdc3c7;
    border-radius: 6px;
    margin-top: 20px; /* Başlık için yer aç */
    font-weight: bold;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* Başlığı ortala */
    padding: 0 10px;
    color: #2c3e50;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="grp_filtre">
         <property name="title">
          <string>TAKİP</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="5" column="0" colspan="2">
           <widget class="QLabel" name="lbl_bilgi">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Özet Bilgi</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QLabel" name="lbl_ozet">
            <property name="frameShape">
             <enum>QFrame::Shape::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cmb_ay">
            <item>
             <property name="text">
              <string>OCAK</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ŞUBAT</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MART</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>NİSAN</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MAYIS</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>HAZİRAN</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>TEMMUZ</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>AĞUSTOS</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>EYLÜL</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>EKİM</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>KASIM</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ARALIK</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="btn_hesapla">
            <property name="text">
             <string>HESAPLA / GETİR</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_yil">
            <property name="text">
             <string>Yıl     </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cmb_yil">
            <item>
             <property name="text">
              <string>2025</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2026</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2027</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2028</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2029</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2030</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_ay">
            <property name="text">
             <string>Ay     </string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="sag_frame">
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTabWidget" name="sekmeli_form">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <property name="elideMode">
          <enum>Qt::TextElideMode::ElideNone</enum>
         </property>
         <widget class="QWidget" name="tab_musteri">
          <attribute name="title">
           <string>Müşteriler</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTableWidget" name="tbl_musteri_hakedis"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_surucu">
          <attribute name="title">
           <string>Sürücüler</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTableWidget" name="tbl_surucu_hakedis"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="footer_frame">
         <property name="frameShape">
          <enum>QFrame::Shape::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Plain</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_excele_aktar">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>-1</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>EXCEL'E AKTAR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_yazdir">
            <property name="text">
             <string>YAZDIR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_geri_don">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>-1</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>GERİ DÖN</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
